<?xml version="1.0" encoding="utf-8"?>
<launch>
    <node name="blattoidea_hw_ros_node" pkg="blattoidea_hw" type="ros_brige_blatoidea_hw.py" output="screen">
        <param name="port" value="/dev/ttyUSB0" />
        <param name="baud" value="115200" />
    </node>
</launch>

